### ISSUE TITLE
addFileFromStream adds empty file

### ISSUE THREAD

**Issue by ivansammartino** [maennchen/ZipStream-PHP#97]
# Description of the problem
When adding files from stream as in the provided example, the zip is created with the file into it, but the file itself is empty

# Example code
After seeing it didn't work on a real example, I tried with the simplest one provided in the documentation (I only changed the `autoload.php` path to suit my needs):

```
require '../vendor/autoload.php';
$zip = new ZipStream\ZipStream('example.zip');

$fp = tmpfile();
fwrite($fp, 'The quick brown fox jumped over the lazy dog.');
$zip->addFileFromStream('goodbye.txt', $fp);
fclose($fp);

$zip->finish();
```
# Informations
More, the CRC32 of goodbye.txt is 00000000, I am not sure this can help solve the issue

* ZipStream-PHP version: 0.5.2
* PHP version: 7.1.20 on Windows (Laragon stack)

**Comment by NicolasCARPi** [comment]
Hello,

That's because the documentation example is bad and doesn't work afaik. (I'll make a PR to fix that).

Here is a working example with stream:

~~~php
require 'vendor/autoload.php';
$zip = new ZipStream\ZipStream('example.zip');
$string = 'The content for the file';
$stream = fopen('php://memory','r+');
fwrite($stream, $string);
rewind($stream);
$zip->addFileFromStream('goodbye.txt', $stream);
$zip->finish();
~~~

**Comment by maennchen** [comment]
Fixed with fb1d6387a3efa263b3fcf810511301e227209c35
